test/features/checkYourAnswers.feature

```diff
@@ -2,6 +2,12 @@
 Feature: Set of scenarios to test functionality of the Check Your Answers page
 
   Background:
+    Given the following definition for 'case fields in events'
+      | Event         | ID             | FieldType       | Display context | Page id         | Page label                    |
```

Comment:
this step configuration doesn't seem relevant to me to the scenario we are testing. No where in the scenario we refer directly to the listed fields, or pages, or page label, or events. 
Either we remove it, or we make it more tailored to the scenario. 
For example we could show in the step that Show Your Answers is enabled in the definition for the field the scenario is using (adding a column 'check your answers' with values yes or no) 

ShowSummaryChangeOption
--

Reply:
I think it is not relevant for the test. It is testing we can change the value and all field are enabled. It would be relevant if we were checking visibility of the field when using `ShowSummaryChangeOption`.